roofit/roofitcore/src/RooFactoryWSTool.cxx

```diff
@@ -938,7 +938,7 @@ std::string RooFactoryWSTool::processExpression(const char* token)
 /// Process a single composite expression
 ///
 /// e.g. 'A=RooGaussian::g[x,m,s]' --> 'A=g'
-/// e.g. 'f[0,1]*RooGaussian::g[x,m,s]' --> 'f*g'
+/// e.g. 'f[0,1]*RooGaussian::%g[x,m,s]' --> 'f*g'
```

Comment:
Can you quickly explain why it's necessary to add the `%` here?

Reply:
to avoid the `g` letter to become a link. See: https://root.cern/doc/master/classRooFactoryWSTool.html#aae005ff5d19079f0bc35a5b1b1541615
But seems to me the whole doc of this function needs to be reformatted. it looks very weird. That `%` is just a hack ... we can do better..

Reply:
I would suggest instead:
```
/// - e.g. `A=RooGaussian::g[x,m,s]` --> `A=g`
/// - e.g. `f[0,1]*RooGaussian::g[x,m,s]` --> `f*g`
/// - e.g. `RooGaussian::g(x,y,s)|x` --> `g|x`
/// - e.g. `$MetaArg(RooGaussian::g[x,m,s],blah)` --> `$MetaArg(g,blah)`
```
which will appear as:

<img width="631" alt="Screenshot 2022-02-10 at 14 49 40" src="https://user-images.githubusercontent.com/4697738/153421082-84a3aa71-f120-4426-84ac-e879fdbc774c.png">

Reply:
changed